kernprof.py

```diff
@@ -437,32 +517,27 @@ def positive_float(value):
     else:
         prof = ContextualProfile()
 
-    # If line_profiler is installed, then overwrite the explicit decorator
-    try:
-        import line_profiler
-    except ImportError:  # Shouldn't happen
-        install_profiler = global_profiler = None
-    else:
-        global_profiler = line_profiler.profile
-        install_profiler = global_profiler._kernprof_overwrite
-
-    if global_profiler:
-        install_profiler(prof)
+    # Overwrite the explicit decorator
+    global_profiler = line_profiler.profile
+    install_profiler = global_profiler._kernprof_overwrite
+    install_profiler(prof)
 
     if options.builtin:
         builtins.__dict__['profile'] = prof
 
     if module:
-        script_file = find_module_script(options.script)
+        script_file = find_module_script(options.script,
+                                         exit_on_error=exit_on_error)
     else:
-        script_file = find_script(options.script)
+        script_file = find_script(options.script, exit_on_error=exit_on_error)
         # Make sure the script's directory is on sys.path instead of
         # just kernprof.py's.
         sys.path.insert(0, os.path.dirname(script_file))
     __file__ = script_file
     __name__ = '__main__'
 
     if options.output_interval:
+        # XXX: why are we doing this here (5a38626) and again below?
```

Comment:
I think we can just remove one of these. I noticed this too, and I think it was a mistake.